docs/mgmt/readme.md

```diff
@@ -0,0 +1,658 @@
+# Management plane SDK generator - how it works and its corresponding configurations
+
+## Resource hierarchy in management plane SDK
+
+For the basic management plane SDK concepts, please refer the detailed document [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/README.md#key-concepts) first.
+
+A resource is usually represented by three classes in the generated SDK: a `[Resource]Data` class, a `[Resource]Resource` class, and a `[Resource]Collection` class.
+
+### `[Resource]Data` class
```

Comment:
Better to use <Resource>Data which is consistent with dotnet guideline.

Reply:
This is referring to the readme of resource manager, which is using `[Resource]` already: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/README.md#L59

Reply:
Oh, and we cannot use <> since it would look like a generic parameter